keps/sig-api-machinery/3285-subresource-finalizers/kep.yaml

```diff
@@ -0,0 +1,52 @@
+title: Subresource finalizers
+kep-number: 3285
+authors:
+  - "@mkimuram"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-api-machinery
+status: provisional
```

Comment:
Change to implementable?

Reply:
> Change to implementable?

@thockin @johnbelamaric Thank you very much for your review. Changed to implementable and squashed the commit.

> I am not the apoprover, (@lavalamp or @apelisse get that honor) but this LGTM

@lavalamp @apelisse PTAL to see if this KEP could be approved.